config.rb

```diff
@@ -39,7 +39,7 @@ def home_url
     if I18n.locale.to_s != 'en' then "/#{I18n.locale}/" else "/" end
   end
   def contact_form_iframe_url
-    "//localhost:5000/#{I18n.locale}/contact/form"
+    ENV.fetch("SWB_CONTACT_FORM_URL", "//localhost:5000/%{locale}/contact/form") % { :locale => I18n.locale }
```

Comment:
Just a tiny thing, but I'd go for the "newer" ruby syntax: `locale: I18n.locale`